storage/innobase/include/ut0new.h

```diff
@@ -1211,4 +1211,36 @@ class aligned_array_pointer : public aligned_memory<T_Type, T_Align_to> {
   size_t m_size;
 };
 
+auto ut_unique_ptr_deleter = [](byte *buf) { ut_free(buf); };
+
+using ut_unique_ptr = std::unique_ptr<byte, decltype(ut_unique_ptr_deleter)>;
+
+inline ut_unique_ptr ut_make_unique_ptr_nokey(const size_t size) {
```

Comment:
Only this variant is required for now. Rest is dead code.

Reply:
@laurynas-biveinis wanted to see those functions for future use